README.md

```diff
@@ -1,112 +1,114 @@
 
-### Requirements
+Note that you first need to build the scripts `FluidSolver` and `SolidSolver`. Each script needs to be build separately.
 
-- [preCICE](https://github.com/precice/precice/wiki/Get-preCICE)
+```
+cd fluid-cpp
+mkdir build && cd build
+cmake ..
+make all
+```
 
-- [LAPACK](http://performance.netlib.org/lapack/#_lapack_version_3_8_0_2). On Ubuntu-like Linux distributions, you can also install via executing:
-  ```bash
-  $ sudo apt-get install liblapack-dev
-  ```
+```
+cd solid-cpp
+mkdir build && cd build
+cmake .. 
+make all
+```
+
+### python
 
-### How to run
+Open two separate terminals and start each participant by calling the respective run script. Only serial run is possible:
 
-0. Clone this repository:
-    ```bash
-    $ git clone https://github.com/precice/elastictube1d.git
-    ```
+```
+python3 ./fluid-python/FluidSolver.py precice-config.xml 
+```
+and
+```
+python3 ./solid-python/SolidSolver.py precice-config.xml 
+```
+Parameters such as N can be modified directly at the `FluidSolver.py` and at the `SolidSolver.py`. The parameters must be coherent between the different solvers and participants. 
 
-1. Navigate into the `cxx` directory and build the Makefiles:
-   ```bash
-   $ cd cxx/ && cmake .
-   ```
-   **Note:** if `cmake` cannot find `libprecice.so`, please make sure that you are [linking to preCICE correctly](https://github.com/precice/precice/wiki/Linking-to-preCICE#linking-from-cmake).
+**Optional:** Visualization and video output of the fluid participant can be triggered via the options `--enable-plot` and `--write-video` of `FluidSolver.py`. To generate .vtk files during execution, you need to add the flag `--write-vtk`.
 
-2. Make the tutorial:
-   ```bash
-   $ make all
-   ```
+## Post-processing
 
-3. After successful compilation, you can now launch preset configuration by calling the `Allrun` script located in the current folder:
-   ```bash
-   $ ./Allrun
-   ```
-   Results will be stored as .vtk files in the `cxx/Postproc` folder.
+The `Postproc/` folder contains the .vtk files resulting from the execution, which you can visualize using eg. paraview. Alternatively you can visualize the results with the provided `Postproc/fluid.py` script:
 
-4. To quickly clean the folder of log files and results from previous runs, execute `Allclean`:
-   ```bash
-   $ ./Allclean
-   ```
-   
-**Optional:** You can visualize the results with the `Postproc/fluid.py` script:
 ```bash
-$ python Postproc/fluid.py <quantity> Postproc/<prefix>
+$ python3 Postproc/fluid.py <quantity> Postproc/<prefix>
 ```
 Note the required arguments specifying which quantity to plot (`pressure`, `velocity` or `diameter`) and a name prefix for the target vtk files.
+
 For example, to plot the diameter using the default prefix for vtk files, we execute:
 ```bash
-$ python Postproc/fluid.py diameter Postproc/out_fluid_
+$ python3 Postproc/fluid.py diameter Postproc/out_fluid_
+```
+![FSI3 setup](images/diameter.png)
```

Comment:
I guess that the image needs to be updated, right?

Reply:
I keep this open in case the updated image is not the final one:)